src/all/akuma/src/eu/kanade/tachiyomi/extension/all/akuma/Akuma.kt

```diff
@@ -176,39 +166,39 @@ class Akuma(
     override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
         val request = popularMangaRequest(page)
 
-        val finalQuery = buildString {
-            append(query)
-            if (lang != "all") {
-                append(" language:", akumaLang, "$")
-            }
-            filters.filterIsInstance<TextFilter>().forEach { filter ->
-                if (filter.state.isBlank()) return@forEach
-                filter.state.split(",").forEach {
-                    // append like `a:"eye-covering bang"$`
-                    if (it.startsWith("-")) {
-                        append(" -", filter.identifier, ":", it.trim().substring(1))
-                    } else {
-                        append(" ", filter.identifier, ":", it.trim())
+        val finalQuery: MutableList<String> = mutableListOf(query)
+
+        if (lang != "all") {
+            finalQuery.add("language: $akumaLang$")
+        }
+        filters.forEach { fIt ->
```

Comment:
```suggestion
        filters.forEach { filter->
```
since fIt is ambigious imo